src/main/scala/higherkindness/rules_scala/workers/zinc/compile/ZincRunner.scala

```diff
@@ -319,7 +312,17 @@ object ZincRunner extends WorkerMain[ZincRunnerWorkerConfig] {
         infos = FilteredInfos.getFilteredInfos(originalResultAnalysis.infos),
       )
     }
-    analysisStoreText.set(AnalysisContents.create(resultAnalysis, compileResult.setup))
+
+    if (verbosity >= 10) {
```

Comment:
I recommend adding a comment here that says verbosity = 10 when `--worker_verbose` is set. Otherwise this may look like an arbitrary constant to folks who don't know that.

Reply:
Good idea.